experiment/process-e2e-suite-log/README.md

```diff
@@ -0,0 +1,63 @@
+# target 
+To process the input file and extract only a given selection of visited endpoints.
+
+## "input file":
+it is a "master log," that is the collection of the endpoints visited while executing a testing run.  
+Here is the [input][input source]: 
+
+This file has an expected and known format: 
+- line 1: "separatorLine".
```

Comment:
What is a separatorLine?

Reply:
"separatorLine" is an arbitrary string that was picked at the moment of the master log generation and separates one complete test execution from another.